pkgs/development/tools/rust/cargo-toml-lint/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib
+, rustPlatform
+, fetchCrate
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-toml-lint";
+  version = "0.1.1";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-U3y9gnFvkqJmyFqRAUQorJQY0iRzAE9UUXzFmgZIyaM=";
+  };
+
+  cargoSha256 = "sha256-ujdekIucqes2Wya4jwTMLstb8JMptbAlqYhgMxfp2gg=";
+
+  meta = with lib; {
+    description = "A simple linter for Cargo.toml manifests";
+    homepage = "https://github.com/fuellabs/cargo-toml-lint";
+    license = with licenses;[ mit asl20 ];
```

Comment:
A pattern that I've seen which can be considered here is to clarify that either license is OK by using:
```
[ asl20 /* or */ mit ]
```

Reply:
I'm curious what other interpretations there are of the list of licenses? E.g. if `license` is a list, are they implicitly `or`d? Is there such thing as `and` licenses?

Reply:
Caveat I'm not an expert, but I don't recall seeing an `and` in the wild. I do think cases exist in which some of the code is under one license and some of the code under another license. I guess I like the clarification because it clears up the ambiguity.

Reply:
I guess my hesitation here is that fwiu these meta fields are more aimed at aiding tooling (like nix-env, the repl, etc), and I'm not sure we should add to the existing PR churn for something like this, particularly when many existing pkgs don't do this.

I appreciate the desire to reduce ambiguity though! I wonder if at some point we can make this more explicit with something like `license = licenses.choiceOf [ asl20 mit  ];` and some more sophisticated `license` field spec in the future.

Reply:
Sounds good. Something like you suggest could be a way to make this explicit structurally.